sample-app/app/models/user.rb

```diff
@@ -1,10 +1,12 @@
 class User < ApplicationRecord
   VALID_EMAIL_REGEX = Settings.email_regex
+  USER_PARAMS = [:name, :email, :password, :password_confirmation]
```

Comment:
```suggestion
  USER_PARAMS = [:name, :email, :password, :password_confirmation].freeze
```
biết freeze để làm gì k e ?

Reply:
Dạ việc sử dụng freeze để không cho thay đổi constant ạ. Vì trong ruby constant có thể thay đổi giá trị được nên cần phải thêm freeze vào ạ